cli/scripts/ace.py

```diff
@@ -676,48 +678,46 @@ def table_diff(
     cluster_name,
     table_name,
     dbname=None,
-    block_rows=1000,
-    max_cpu_ratio=MAX_CPU_RATIO,
+    block_rows=BLOCK_ROWS_DEFAULT,
+    max_cpu_ratio=MAX_CPU_RATIO_DEFAULT,
     output="json",
     nodes="all",
     diff_file=None,
 ):
     """Efficiently compare tables across cluster using checksums and blocks of rows"""
-
-    if not diff_file:
+    
+    if type(block_rows) is str:
         try:
-            block_rows = int(os.environ.get("ACE_BLOCK_ROWS", block_rows))
+            block_rows = int(block_rows)
         except Exception:
             util.exit_message("Invalid values for ACE_BLOCK_ROWS")
-    try:
-        max_cpu_ratio = float(os.environ.get("ACE_MAX_CPU_RATIO", max_cpu_ratio))
-    except Exception:
-        util.exit_message("Invalid values for ACE_MAX_CPU_RATIO")
-
-    if max_cpu_ratio > 1 or max_cpu_ratio < 0:
-        util.exit_message("Invalid values for ACE_MAX_CPU_RATIO or --max_cpu_ratio")
-
-    max_cpu_ratio = float(max_cpu_ratio)
-
+    elif type(block_rows) is not int:
+        util.exit_message("Invalid value type for ACE_BLOCK_ROWS")
+    
     # Capping max block size here to prevent the hash function from taking forever
     if block_rows > MAX_ALLOWED_BLOCK_SIZE:
         util.exit_message(f"Desired block row size is > {MAX_ALLOWED_BLOCK_SIZE}")
+    if block_rows < MIN_ALLOWED_BLOCK_SIZE:
+        util.exit_message(f"Desired block row size is < {MIN_ALLOWED_BLOCK_SIZE}")
+
+    if type(max_cpu_ratio) is int:
```

Comment:
Do you need checks for int, str, etc? You could cast it to float inside a try block, and on ValueError, you could have a util.exit_message. After this, you could check for the range.